resources/configurationFiles/analysisRNAseq.xml

```diff
@@ -71,7 +71,7 @@
         <cvalue name='RNASEQC_VERSION' value='1.1.8' type='string'/>
         <cvalue name='KALLISTO_VERSION' value='0.43.0' type='string'/>
         <cvalue name='QUALIMAP_VERSION' value='2.2.1' type='string'/>
-        <cvalue name='ARRIBA_VERSION' value='1.2.0' type='string'/>
+        <cvalue name='ARRIBA_VERSION' value='2.2.1' type='string'/>
```

Comment:
I am preparing a release and noticed you didn't update STAR for the old assemblies. Was a newer star not necessary for arriba 2.2.1?

Reply:
It's not necessary for the LSF modules. Arriba is backwards-compatible to old STAR versions. Only conda requires that STAR must be updated to 2.7.10a, because the Arriba 2.2.1 conda package has STAR 2.7.10a as a dependency.

That being said, it is recommended to update to the latest STAR version.

Reply:
Hm. Maybe we should update to 2.7.10a as default STAR version then for all assemblies. But we'd also have to make sure that the required indices are being installed locally and of course the conda.yaml to be updated.

Reply:
Ha, there is no conda.yaml! Maybe you could add your war-proven Conda environment YAML to the upcoming pre-release-4 PR later?

Reply:
I don't have a conda.yaml. I used the LSF modules for testing. There is a `environment.yaml`, which needs updating. But I don't know how it was created. This is what I asked about in my pull request description.

Reply:
Yes, I see. In some other workflows I put the YAML into environment/ directory somewhere under resources/. 

I think that the conda environment.yaml should be updated. I asked you because I though you might have updated it anyway (locally; e.g. for your containers), but if that is not the case, I can update it. I'm going to add to the pre-release-4 branch.